apps/fxc-front/src/app/redux/track-slice.ts

```diff
@@ -44,6 +46,11 @@ export type TrackState = {
   loaded: boolean;
 };
 
+export type RuntimeTrackId = Pick<RuntimeTrack, 'id'>
```

Comment:
suggest: move that type together with the declaration of RuntimeTrack?

Reply:
I removed this type as it was just for a single simplification of code and thus not really useful